packages/web/src/index.ts

```diff
@@ -4,3 +4,4 @@ export * from './transports';
 export * from './config';
 export * from './initialize';
 export { LogLevel } from '@grafana/agent-core';
+export type { Agent } from '@grafana/agent-core';
```

Comment:
I think we also need to export the actual `agent` object.

Reply:
The rationale behind this is: if you manually install `@grafana/agent-core` and import the agent from there, that would be another instance of the agent instead of the one used by `@grafana/agent-web`.